components/RTM/RealTimeMonitor.vue

```diff
@@ -33,6 +33,24 @@
         <p v-show="!loading && actives.length === 0 && !error" class="mt-2">
           There are no alerts currently running!
         </p>
+        <div v-show="owactives.length > 0">
+          <div
+            v-for="outfitwar in owactives"
+            :key="outfitwar.instanceId"
+            class="p-1 border-b border-gray-500 border-no-bottom"
+          >
+            <RealTimeOutfitWar
+              :world="outfitwar.world"
+              :zone="outfitwar.zone"
+              :time-started="outfitwar.timeStarted"
+              :duration="outfitwar.duration"
+              :result="outfitwar.result"
+              :instance-id="outfitwar.instanceId"
+              :pops="getPops(outfitwar.instanceId)"
```

Comment:
This likely won't work at all yet, aggregator needs changing to detect these pops on the instanced zones